crates/bws/src/main.rs

```diff
@@ -388,26 +388,23 @@ async fn process_commands() -> Result<()> {
         | Commands::Delete {
             cmd: DeleteCommand::Project { project_ids },
         } => {
+            let mut projects_success = project_ids.len();
```

Comment:
We should ideally try and avoid mutable variables. In this case we can just assign the variable on line 403 directly?

Reply:
Because `Vec` doesn't implement the Copy trait, the value is moved on line 395 when `.delete(ProjectsDeleteRequest { ids: project_ids })` is called, so I need to capture the value before it's moved. I see three options, please let me know what you think!
- We could change the `delete` params to accept a reference to `ProjectsDeleteRequest`, but I figure that would be outside the scope of this PR.
- I can just use variable shadowing and re-declare `projects_success`
- Something else / leave as is

Reply:
So you want something like the following:

`let count = project_ids.len();`

Then later on line 562: `let deleted_projects = count - projects_failed.len();`

Reply:
Yes. That gets rid of the mutable variables which are bad.

Reply:
Refactored in [daf7921](https://github.com/bitwarden/sdk/pull/77/commits/daf7921b6c4bba6d97328e0bd113c7fa077319af)